apps/developer-portal/src/components/claims/edit/external-dialect/edit-external-claims.tsx

```diff
@@ -77,12 +75,13 @@ export const EditExternalClaims = (props: EditExternalClaimsPropsInterface): Rea
     ];
 
     const [ offset, setOffset ] = useState(0);
-    const [ listItemLimit, setListItemLimit ] = useState<number>(0);
+    const [ listItemLimit, setListItemLimit ] = useState<number>(UIConstants.DEFAULT_RESOURCE_LIST_ITEM_LIMIT);
     const [ filteredClaims, setFilteredClaims ] = useState<ExternalClaim[]>([]);
     const [ sortBy, setSortBy ] = useState(SORT_BY[ 0 ]);
     const [ sortOrder, setSortOrder ] = useState(true);
     const [ showAddExternalClaim, setShowAddExternalClaim ] = useState(false);
-    const [ query, setQuery ] = useState("");
+    const [ searchQuery, setSearchQuery ] = useState("");
```

Comment:
Let add the type of the state variable here.

Reply:
Fixed with `f0c0e3f`